nbs/examples/Exogenous.ipynb

```diff
@@ -7,58 +7,111 @@
    "source": [
```

Comment:
<p>here could we include a more detailed description of how to include exogenous variables? Something like,</p><p> </p><pre class="ql-syntax" spellcheck="false">statsforecast considers as exogenous variables all columns after the column <code>y</code>
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Done. </p>